gflib/text.h

```diff
@@ -354,10 +354,10 @@ struct KeypadIcon
 };
 
 typedef struct {
-    bool8 canABSpeedUpPrint:1;
-    bool8 useAlternateDownArrow:1;
-    bool8 autoScroll:1;
-    bool8 forceMidTextSpeed:1;
+    bool8 canABSpeedUpPrint:TRUE;
+    bool8 useAlternateDownArrow:TRUE;
+    bool8 autoScroll:TRUE;
+    bool8 forceMidTextSpeed:TRUE;
```

Comment:
These are numbers of bits, not boolean values

Reply:
the usage of these variables say otherwise, but ok. are you sure?

Reply:
The `:1` indicates these fields are 1 bit in size. They're bools by definition of being 1 bit, but these 1's don't have anything to do with truth values

Reply:
Ok